modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/containers/setup-forms/SetupDXPCloudForm/index.js

```diff
@@ -158,79 +162,107 @@ const SetupDXPCloudPage = ({
 				title: 'Set up DXP Cloud',
 			}}
 		>
-			<div className="d-flex justify-content-between mb-2 pb-1 pl-3">
-				<div className="mr-4 pr-2">
-					<label>Project Name</label>
+			<FieldArray
+				name="dxp.admins"
+				render={({pop, push}) => (
+					<>
+						<div className="d-flex justify-content-between mb-2 pb-1 pl-3">
+							<div className="mr-4 pr-2">
+								<label>Project Name</label>
 
-					<p className="dxp-cloud-project-name text-neutral-6 text-paragraph-lg">
-						<strong>
-							{project.name.length > 71
-								? project.name.substring(0, 71) + '...'
-								: project.name}
-						</strong>
-					</p>
-				</div>
+								<p className="dxp-cloud-project-name text-neutral-6 text-paragraph-lg">
+									<strong>
+										{project.name.length > 71
+											? project.name.substring(0, 71) +
```

Comment:
I think would be better use constants instead magic numbers

https://pt.wikipedia.org/wiki/N%C3%BAmero_m%C3%A1gico_(programa%C3%A7%C3%A3o_de_sistemas)

https://stackoverflow.com/questions/47882/what-is-a-magic-number-and-why-is-it-bad